functions-app/school-import/service/data-access/predicates.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+const RA = require('ramda-adjunct')
+
+const predicates = {
+  isSchoolOpen (logger, school) {
+    // we want to load all schools that are open, proposed to open, proposed to close
+    // this is the same as every school that isn't closed
+    if (school.estabStatusCode === '2') {
```

Comment:
magic number? perhaps introduce a const to make this if statement a bit more readable?

Reply:
Good spot.  Updated.